src/libopensc/reader-pcsc.c

```diff
@@ -2318,6 +2343,9 @@ int cardmod_use_reader(sc_context_t *ctx, void * pcsc_context_handle, void * pcs
 		}
 		priv->gpriv = gpriv;
 
+		reader->max_send_size = PCSC_SHORT_APDU_MAX_SEND_SIZE;
```

Comment:
The value from `opensc.conf` or from the reader's tlv is not read on Windows? It should be done!